nautobot/core/testing/views.py

```diff
@@ -529,13 +529,32 @@ def test_delete_object_without_permission(self):
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
         def test_delete_object_with_permission(self):
+            from nautobot.extras.models import Note, ObjectChange
+
             instance = self.get_deletable_object()
+            instance_note_pk_list = []
+            assigned_object_type = ContentType.objects.get_for_model(self.model)
+            if hasattr(self.model, "notes"):
```

Comment:
Probably should add `isinstance(instance.notes, NotesQuerySet)` here and below to be safe in case of apps using these generic tests with their own `notes` fields?